src/ui/src/js/services/garden_service.js

```diff
@@ -307,6 +387,12 @@ export default function gardenService($http) {
                 'stomp.username',
                 'stomp.password',
                 'stomp.ssl',
+                {
```

Comment:
`angular-schema-forms` is abandonware. It hasn't been updated in 5 years and it has outstanding PRs from 7 years ago. It has problems. At any rate, many cycles were wasted trying to get the form to properly update on the page when the data was updated, all for nothing. The issue appears to be with the library itself. This message to the user to refresh the page is acknowledgement of defeat. :(